src/nbl/ext/ToneMapper/CToneMapper.cpp

```diff
@@ -193,7 +201,7 @@ core::smart_refctd_ptr<ICPUSpecializedShader> CToneMapper::createShader(
 
 	const char* usingTemporalAdaptationDefine = usingTemporalAdaptation ? "#define _NBL_GLSL_EXT_TONE_MAPPER_USING_TEMPORAL_ADAPTATION_DEFINED_":"";
 
-	constexpr char* sourceFmt =
+	const char* sourceFmt =
```

Comment:
Turn into `nbl/builtin/glsl/ext/ToneMapper/default.comp`, override like I told you for LumaMeter


P.S. in the interest of legibility you should probably forward declare `nbl_glsl_ext_ToneMapper_readColor` and `nbl_glsl_ext_ToneMapper_writeColor`